pkg/blobaccess/file/access.go

```diff
@@ -0,0 +1,338 @@
+package file
+
+import (
+	"io"
+	"sync"
+
+	"github.com/mandelsoft/goutils/errors"
+	"github.com/mandelsoft/vfs/pkg/vfs"
+	"github.com/opencontainers/go-digest"
+
+	"github.com/open-component-model/ocm/pkg/blobaccess/bpi"
+	"github.com/open-component-model/ocm/pkg/iotools"
+	"github.com/open-component-model/ocm/pkg/utils"
+)
+
+type (
+	_nopCloser  = iotools.NopCloser
```

Comment:
What's the benefit of redeclaring types? Doesn't it hide the usage of `iotools.NopCloser` and make refactorings more complex?

Reply:
It makes is private for the embedding types. But here it is not really necessary,
, because it does not have fields.